src/main/java/org/jenkinsci/plugins/parallel_test_executor/ParallelTestExecutor.java

```diff
@@ -155,14 +158,41 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         return true;
     }
 
+    public static Map<String, TestClass>  findTestResultsInDirectory(Run<?,?> build, TaskListener listener, FilePath workspace){
```

Comment:
method name is confusing as it is not finding test results but test source.

Reply:
yes you are right, my bad .... I will rename it.